File: Upcoder/Other/Xuất vị trí chứa giá trị âm lớn nhất trong mảng/LTCB-154.py
n = int(input())
arr = list(map(float, input().split()))

min_val = -99999999
pos = -1

for i in range(n):
    if arr[i] < 0 and (pos == -1 or arr[i] < min_val):
        min_val = arr[i]
        pos = i

print(pos)

File: Upcoder/Other/Xuất vị trí chứa giá trị âm lớn nhất trong mảng/LTCB-154.cs
using System;
namespace MinAm
{
    public class Program
    {
        static void Main(string[]args)
        {
            int n = int.Parse(Console.ReadLine());
            string []input = Console.ReadLine().Split(' ');
            double [] Arr = new double[n];
            for(int i = 0; i < n; i++)
                Arr[i] = double.Parse(input[i]);
            double Min = -99999999;
            int pos = -1;
            for(int i = 0; i < n; i++) {
                if (Arr[i] < 0 && (pos == -1 || Arr[i] < Min)) 
                {
                    Min = Arr[i];
                    pos = i;
                }
            }
            Console.Write(pos);
        }
    }
}